tests/components/apprise/test_notify.py

```diff
@@ -0,0 +1,107 @@
+"""The tests for the apprise notification platform."""
+import os
+import unittest
+from unittest.mock import patch
+from tempfile import mkdtemp
+
+from apprise import Apprise
+from apprise import AppriseConfig
+
+from homeassistant.setup import setup_component
+import homeassistant.components.notify as notify
+from tests.common import assert_setup_component, get_test_home_assistant
+
+
+class TestApprise(unittest.TestCase):
```

Comment:
New tests must be standalone pytest test functions. See eg:

https://github.com/home-assistant/home-assistant/blob/1c72a246a048d0f2f3d3dca0b77be559dae01a78/tests/components/manual/test_alarm_control_panel.py#L83-L119

Reply:
@MartinHjelmare, first off: thank you for your help offline.  I was able to build onto the tests and get much further as a result.  I'm still hitting a wall, but I do see a light at the end of the tunnel at least.

The test that is failing is: [test_apprise_notification()](https://github.com/home-assistant/home-assistant/blob/cdf482eb88eb9e3d556e56b3893fcd58edb3985b/tests/components/apprise/test_notify.py#L52).  It appears that `hass.services.async_call()` is causing the tests to abort.

The error:
```
E    homeassistant.exceptions.ServiceNotFound: Unable to find service notify/test

../../homeassistant/core.py:1209: ServiceNotFound
```

Your thoughts/advice would be most appreciated (or anyone for that matter reviewing this). :slightly_smiling_face:

Reply:
We need to set up the correct component. See my change suggestions below.